bazel/external/icu/data/udata.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/icu4c/source/common/udata.cpp b/icu4c/source/common/udata.cpp
```

Comment:
Add comment what this patch do and why it is needed.

Reply:
Comment added.